doc/release/0.3.rst

```diff
@@ -14,7 +14,7 @@ New Features
 #. Reset ``Reactor`` data on worker processors after every interaction to free memory from state distribution.
    (`PR#1729 <https://github.com/terrapower/armi/pull/1729>`_ and `PR#1750 <https://github.com/terrapower/armi/pull/1750>`_)
 #. Density can be specified for components via ``custom isotopics`` in the blueprints. (`PR#1745 <https://github.com/terrapower/armi/pull/1745>`_)
-#. Implement a new `JaggedArray` class that handles HDF5 interface for jagged data. (`PR#1726 <https://github.com/terrapower/armi/pull/1726>`_)
+#. Implement a new `JaggedArray` class that handles HDF5 interface for jagged data. (`PR#1726 <https://github.com/terrapower/armi/pull/1726>`_, `PR#1778 <https://github.com/terrapower/armi/pull/1778>`_)
```

Comment:
```suggestion
#. Implement a new `JaggedArray` class that handles HDF5 interface for jagged data. (`PR#1778 <https://github.com/terrapower/armi/pull/1726>`_, `PR#1778 <https://github.com/terrapower/armi/pull/1778>`_)
```

Reply:
Actually, I'm not sure we need to change the release note at all. The first PR was the more important one.